src/docs/guides/phoenix.md

```diff
@@ -0,0 +1,174 @@
+
+1. [One-click deploy from a template](#one-click-deploy-from-a-template).
+2. [Using the CLI](#deploy-from-the-cli).
+3. [From a GitHub repository](#deploy-from-a-github-repo).
+
+## One-Click Deploy from a Template
+
+If you’re looking for the fastest way to get started, the one-click deploy option is ideal. It sets up a Phoenix app along with a Postgres database.
+
+Click the button below to begin:
+
+[![Deploy on Railway](https://railway.app/button.svg)](https://railway.app/new/template/0LSBzw)
+
+After deploying, we recommend that you [eject from the template](/guides/deploy#eject-from-template-repository) to create a copy of the repository under your own GitHub account. This will give you full control over the source code and project.
+
+## Deploy from the CLI
+
+To deploy the Phoenix app using the Railway CLI, please follow the steps:
+
+1. **Install the Railway CLI**:
+    - <a href="/guides/cli#installing-the-cli" target="_blank">Install the CLI</a> and <a href="/guides/cli#authenticating-with-the-cli" target="_blank">authenticate it</a> using your Railway account.
+2. **Initialize a Railway Project**:
+    - Run the command below in your Phoenix app directory. 
+        ```bash
+        railway init
+        ```
+    - Follow the prompts to name your project.
+    - After the project is created, click the provided link to view it in your browser.
+3. **Deploy the Application**:
+    - Use the command below to deploy your app:
+        ```bash
+        railway up
+        ```
+    - This command will scan, compress and upload your app's files to Railway. You’ll see real-time deployment logs in your terminal.
+
+    **Note:** You might encounter an error––`warning: the VM is running with native name encoding of latin1 which` `may cause Elixir to malfunction as it expects utf8....`. Don’t worry, we’ll address this in the next step.
+4. **Add a Database Service**:
+    - Run `railway add`.
+    - Select `PostgreSQL` by pressing space and hit **Enter** to add it to your project.
+    - A database service will be added to your Railway project.
+5. **Configure Environment Variables**:
+    - Go to your app service <a href="/overview/the-basics#service-variables">**Variables**</a> section and add the following:
+        - `SECRET_KEY_BASE` : Set the value to the your app's secret key. You can run `mix phx.gen.secret` locally to generate one.
+        - `LANG`and `LC_CTYPE`: Set both values to `en_US.UTF-8`. This sets your app's locale and gets rid of the _native name encoding of latin1_ warning.
+        - `DATABASE_URL`: Set the value to `${{Postgres.DATABASE_URL}}` (this references the URL of your new Postgres database). Learn more about [referencing service variables](/guides/variables#referencing-another-services-variable).  
+        - `ECTO_IPV6`: Set the value to `true` to enable your Phoenix app to connect to the database through the IPv6 private network. 
```

Comment:
Maybe link to our guide on the private network so the user can learn why IPv6 is used in the first place.

Reply:
Good idea.

Reply:
Fixed.